fido-device-onboard.spec

```diff
@@ -11,7 +11,9 @@ License:        BSD-3-Clause
 URL:            https://github.com/fdo-rs/fido-device-onboard-rs
 Source0:        %{url}/archive/v%{version}/%{name}-rs-%{version}.tar.gz
 # See make-vendored-tarfile.sh in upstream repo
+%if 0%{?el}
```

Comment:
Should be 0%{?rhel}

Reply:
already tried and doesn't work either, will keep trying

Reply:
Yes, I also think the changes that landed yesterday were incorrect but they were merged before I could comment.

Reply:
why is that? it's working besides a conditional for centos